src/App.jsx

```diff
@@ -9,32 +10,19 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
Follow the task description, please 
![image](https://github.com/mate-academy/react_tabs-js/assets/99403198/d8eaeea8-8cd7-4632-9ba2-3fff673b68ed)